pandas/tests/extension/json/test_json.py

```diff
@@ -130,7 +132,11 @@ def test_custom_asserts(self):
 
 
 class TestConstructors(BaseJSON, base.BaseConstructorsTests):
-    pass
+
+    @pytest.mark.xfail(reason="not implemented constructor from dtype")
+    def test_from_dtype(self, data):
+        # construct from our dtype & string dtype
+        pass
```

Comment:
Can we test here that an appropriate error is raised (instead of skipping the test) ?